client/src/pages/Details.js

```diff
@@ -76,20 +97,24 @@ function Details() {
                       sint occaecat cupidatat non proident, sunt in culpa qui
                       officia deserunt mollit anim id est laborum.
                     </p>{" "}
-                    <Button
-                      variant="outlined"
-                      startIcon={<DeleteIcon />}
-                      style={{
-                        color: "white",
-                        backgroundColor: "red",
-                        padding: "10px",
-                        cursor: "pointer",
-                        borderRadius: "5px",
-                      }}
-                      onClick={() => handleDelete(data.id)}
-                    >
-                      Delete
-                    </Button>
+                    {popup.show && (
+                      <Button
+                        variant="outlined"
+                        startIcon={<DeleteIcon />}
+                        style={{
+                          color: "white",
+                          backgroundColor: "red",
+                          padding: "10px",
+                          cursor: "pointer",
+                          borderRadius: "5px",
+                        }}
```

Comment:
I know it wasn't introduced in this PR, but it would be great to see these inline styles converted to something more sustainable and scalable like a CSS file or the Mui styling syntax.

Reply:
Doing some research ...tried styled components and that didn't work all the way.

Reply:
`styled-components` _should_ work here.  What selector(s) are you using?

Reply:
it will get pulled in once I merge in the other PR